modules/post-installation-adding-nutanix-failure-domains-compute-machines.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-compute-machines_{context}"]
+= Distributing compute machines across failure domains
+
+You distribute compute machines across Nutanix failure domains by modifying the control plane machine set.
+
+.Prerequisites
+
+* You have added the failure domains to the cluster's Infrastructure custom resource definition (CRD).
+
+.Procedure
+
+. List the compute machine sets in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
+. Edit the first compute machine set by running the following command:
++
+[source,terminal]
+----
+$ oc edit machineset <machineset_name> -n openshift-machine-api
+----
+. Note the value of `spec.replicas`, as you need it when scaling up the machine set to apply the changes.
+. Configure the compute machine set to use the the first failure domain by adding the following to the `spec.template.spec.providerSpec.value` stanza:
++
+.Example compute machine set with Nutanix failure domains
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1
+kind: MachineSet
+metadata:
+  creationTimestamp: null
+  labels:
+    machine.openshift.io/cluster-api-cluster: <cluster_name>
+  name: <machineset_name>
+  namespace: openshift-machine-api
+spec:
+  replicas: 2
+# ...
+  template:
+    spec:
+# ...
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1
+          failureDomain:
+            name: <failure_domain_name_1>
+          cluster:
+            type: uuid
+            uuid: <prism_element_uuid_1>
+          subnets:
+          - type: uuid
+            uuid: <prism_element_subnet_uuid_1>
+# ...
+----
+. Save your changes.
+. Scale down the compute machine set to `0` by running the following command:
```

Comment:
That's what we have in [docs](https://docs.openshift.com/container-platform/4.14/machine_management/modifying-machineset.html) for any change to machine sets, currently. Quite possible it's outdated; is there a better method to recommend or is it somehow handled automagically now?

Reply:
Closing this out: Joel suggested a better way to do this. I will update the content at [Modifying a compute machine set](https://docs.openshift.com/container-platform/4.14/machine_management/modifying-machineset.html#machineset-modifying_modifying-machineset), and then we can use those updates for this procedure. This can happen in parallel and shouldn't hold up this PR.